pages/wiki/update-asteroidos.md

```diff
@@ -9,11 +9,11 @@ As currently no AsteroidOS watch has access to the internet by stock means, no G
 
 # Upgrade using OPKG
 
-*****
+The first step in using `opkg` is to make sure the time and date are set correctly.  That's required so that the watch can correctly validate the remote server's certificate and establish a secure connection.
```

Comment:
Maybe a bit shorter and put the instruction first. Correctly is used two times. The "so" and "That's" are colloquialisms we can avoid in docs imo.

"Make sure the time and date are set correctly before using `opkg`. Else the watch can not validate the remote server's certificate and establish a secure connection".

Reply:
Great, even shorter! But now it could mean to set the time and date on the host (with a lot of stretch and noob factor)?
How about
"Set the correct time and date on the watch. AsteroidOS needs to validate the remote server's certificate to establish a secure connection."

Reply:
Perfect.  That's what I'll use.

Reply:
It took us 20 minutes to polish that sentence. But with thousands of users now reading one second less. We made the world a little tad better :rofl: 
Not to think of the massive time saver compared to when all those would have run in the issue and ask for help.
Again, nice work to pick this up.